### PR TITLE
Feature/character card renders

### PR DESCRIPTION
**Pull request by rmuehler**
1. Small refactore for old test to use character array from Character.json
2. New test for rendering name and image for Character.

### CODE REVIEW

**Comment by arthuranteater** [comment]
err in karma test

File: src/app/components/character-list/character-list.component.spec.ts [hunk 1]
```diff
@@ -1,14 +1,14 @@
 import { ComponentFixture, TestBed } from '@angular/core/testing';
 
 import { CharacterListComponent } from './character-list.component';
+import  Character  from '../../models/character';
```
Comment (stevenpazur) [inline src/app/components/character-list/character-list.component.spec.ts]: Should be import { Character } from '../../models/character';

**Review by stevenpazur — COMMENTED** [review]
Add brackeys around Character on line 4 of character-list spec file.

File: src/app/components/character-list/character-list.component.spec.ts (hunk 1 above)
Comment (rmuehler) [reply src/app/components/character-list/character-list.component.spec.ts]: Good catch.

Comment (LWooler) [reply src/app/components/character-list/character-list.component.spec.ts]: Import from interface, not class. Import should be correct without brackets

**Comment by rmuehler** [comment]
Should be good to go now, we had an import problem.

**Pull request merged**